src/condor_dagman/dagman_submit.cpp

```diff
@@ -358,10 +358,11 @@ static bool direct_condor_submitV2(const Dagman &dm, Node* node, CondorID& condo
 	std::string errmsg;
 	std::string URL;
 	auto_free_ptr owner(my_username());
+	std::string_view& inline_desc = node->GetInlineDesc();
```

Comment:
`std::string_views` are small enough (two pointers) that they usually fit in a pair of registers, so it often is better to copy them around by value instead of by reference.  Also, could `inline_desc` be const?